interface/interface.dm

```diff
@@ -157,5 +173,5 @@ Admin:
 
 	src << hotkey_mode
 	src << other
-	if(holder)
+	if(check_rights(R_ADMIN, FALSE))
 		src << admin
```

Comment:
Undocumented unrelated change.

Please remove this change and move it to its own pr.

Reply:
If it's a simple file I'm touching I usually clean it up, but sure.